### ISSUE TITLE
taxonomic names in Synospecies

### ISSUE THREAD

**Issue by myrmoteras** [factsmission/synospecies#178]
The representation of taxonomic names on the left side in the taxon display in [Synospecies](https://synospecies.plazi.org/#Tyrannosaurus+rex) is unusual and not according taxonomic practice.

![image](https://github.com/user-attachments/assets/fc1c70e7-80c3-465f-a81b-48c271a55604)

e.g. Nanotyrannus lancensis (Gilmore, 1946) / Gilmore, 1946 / (Gilmore, 1946) Bakker et al., 1988 / (Gilmore, 1946) Bakker, Currie & Williams, 1988

in Zoology, the name should be Nanotyrannus lancensis (Gilmore, 1946), whereby the change of generic name is indictated by a bracket.

in Botany, the change is indicated by the authority that made the change.

currently, the name includes all the nomenclatural changes. 

@millerjeremya suggests to display the names according to taxonomic rules. 

A solution could be to add a comment explaining the unique display of names?

**Comment by millerjeremya** [comment]
I don't think it makes any sense to employ a unique authority system on a platform that is intended to resolve issues of taxonomic nomenclature. The authority is a part of the species name, so by introducing something novel, synospecies misses its core purpose of mapping relationships among treatments based on citations. The synospecies authority system is not explained, and is not needed. 
See also this related issue:
https://github.com/plazi/names_LOD/issues/154

**Comment by nleanba** [comment]
I agree that the current solution is unsatisfactory. However, I am not sure on how to fix it.

The issue is that Synospecies is not "displaying all nomenclatural changes", but rather that each authority (separated by "/") is a separate entry in the data, stemming from different treatments.

e.g. there is a treatment which says "Genus species Name A 1234" and one that says "Genus species (Name B 1324)" and Synospecies has no way to decide which one of these is correct/current.

How should Synospecies (or ideally gg2rdf) decide which of the authorities to show? Open to suggestions

**Comment by nleanba** [comment]
Impossible or not, it occurs in the data:
- Treatment 0F0287A7FFA3FFC97375FEE7FE1DF91D:

	https://git.ld.plazi.org/plazi/treatments-xml/src/commit/51f1c6b70ef9d3d46f028f6977e0992f46e7c12d/data/0F/02/87/0F0287A7FFA3FFC97375FEE7FE1DF91D.xml#L42
	```xml
	<taxonomicName baseAuthorityName="Gilmore" baseAuthorityYear="1946" ...>Nanotyrannus lancensis</taxonomicName>
	```
	gets translated into rdf as:
	```ttl
	<http://taxon-concept.plazi.org/id/Animalia/Nanotyrannus_lancensis_Gilmore_1946>
	    dwc:baseAuthorityName "Gilmore" ;
	    dwc:baseAuthorityYear "1946" ;
	    dwc:scientificNameAuthorship "(Gilmore, 1946)" .
    ```

- Treatment 03A1879DFFD0FFE95E7FF664FEA6A618:

    https://git.ld.plazi.org/plazi/treatments-xml/src/commit/51f1c6b70ef9d3d46f028f6977e0992f46e7c12d/data/03/A1/87/03A1879DFFD0FFE95E7FF664FEA6A618.xml#L55
	```xml
	<taxonomicName authorityName="Gilmore" authorityYear="1946" ...>Nanotyrannus</taxonomicName>
	```
	→
	```ttl
	<http://taxon-concept.plazi.org/id/Animalia/Nanotyrannus_lancensis_Gilmore_1946>
	    dwc:authorityName "Gilmore" ;
	    dwc:authorityYear "1946" ;
	    dwc:scientificNameAuthorship "Gilmore, 1946" .
	```
	
- Treatment B97187EFFFC24A13FCE4FD5B7F0DFEAC:

    https://git.ld.plazi.org/plazi/treatments-xml/src/commit/51f1c6b70ef9d3d46f028f6977e0992f46e7c12d/data/B9/71/87/B97187EFFFC24A13FCE4FD5B7F0DFEAC.xml#L45
    ```xml
    <taxonomicName authorityName="Bakker et al." authorityYear="1988" baseAuthorityName="Gilmore" baseAuthorityYear="1946" ...>Nanotyrannus lancensis</taxonomicName>
    ```
    →
    ```ttl
    <http://taxon-concept.plazi.org/id/Animalia/Nanotyrannus_lancensis_Gilmore_1946>
	    dwc:authorityName "Bakker et al." ;
	    dwc:authorityYear "1988" ;
	    dwc:baseAuthorityName "Gilmore" ;
	    dwc:baseAuthorityYear "1946" ;
	    dwc:scientificNameAuthorship "(Gilmore, 1946) Bakker et al., 1988" .
    ```
	
- Treatment 03E8486CFFE30E52FFBE5400FBC3FE74:


    https://git.ld.plazi.org/plazi/treatments-xml/src/commit/51f1c6b70ef9d3d46f028f6977e0992f46e7c12d/data/B9/71/87/B97187EFFFC24A13FCE4FD5B7F0DFEAC.xml#L45
    ```xml
    <<taxonomicName authorityName="Bakker, Currie &amp; Williams" authorityYear="1988" baseAuthorityName="Gilmore" baseAuthorityYear="1946" ...>Nanotyrannus</taxonomicName>
    ```
    →
    ```ttl
    <http://taxon-concept.plazi.org/id/Animalia/Nanotyrannus_lancensis_Gilmore_1946>
	    dwc:authorityName "Bakker, Currie & Williams" ;
	    dwc:authorityYear "1988" ;
	    dwc:baseAuthorityName "Gilmore" ;
	    dwc:baseAuthorityYear "1946" ;
	    dwc:scientificNameAuthorship "(Gilmore, 1946) Bakker, Currie & Williams, 1988" .
    ```

I admit the latter two examples are bad examples, as for Animalia gg2rdf should ignore the authority if a baseAuthority is given, I will fix this. However, the general problem still exists, and this alos happens for non-Animialia taxa, so situations like with the latter two treatments should still be handled properly.

My point is, those four treatments considered in isolation (and assuming non-Animalia) all are apparently correct translations of the XML markup into RDF. No one of the treaments alone contains the information that there is a mismatch and what authority is the correct one.

**Comment by millerjeremya** [comment]
Thanks for this example. I think it's a useful approach to work through. 
So the thing that jumps out at me has to do with Treatment 03A1879DFFD0FFE95E7FF664FEA6A618
You provide these annotations in the example

 <taxonomicName authorityName="Gilmore" authorityYear="1946" ...>Nanotyrannus</taxonomicName>
→

 <http://taxon-concept.plazi.org/id/Animalia/Nanotyrannus_lancensis_Gilmore_1946>
     dwc:authorityName "Gilmore" ;
     dwc:authorityYear "1946" ;
     dwc:scientificNameAuthorship "Gilmore, 1946" .


This was indeed a tricky one to annotate. It is a genus level treatment (new genus) but of a monotypic genus, so it works kind of like a species level treatment. So I'm changing the authorityName and authorityYear annotations into baseAuthorityName and baseAuthorityYear annotations. How does that effect the result on Synospecies? 

![image](https://github.com/user-attachments/assets/5b7073ca-c80e-414c-b866-5bbd0f353842)
![image](https://github.com/user-attachments/assets/d30ecdef-ca48-4452-9514-e17eb72c7a34)

**Comment by nleanba** [comment]
If it is annotated as _base_ authority, it will receive the parentheses. So this should fix this particular example

**Comment by millerjeremya** [comment]
Ok, so how do we find sources of conflict in the authority? 
For example, 
Tyrannosaurus rex (Osborn, 1905) / Osborn, 1905
If I understand correctly, this indicates that in at least one place, Osborn is assigned to the field baseAuthorityName instead of AuthorityName. 
But as I scan down the list of treatments, all give the authority as though is is AuthorityName, i.e., no parentheses. 

![image](https://github.com/user-attachments/assets/565236b0-885f-4910-a477-e9a3d266eb22)

**Comment by myrmoteras** [comment]
Can we discuss Thursday at 2pm?

New species of the day<https://tb.plazi.org/GgServer/static/newToday.html>
Plazi<https://plazi.org/>
Bouchout+10 symposium<https://bouchoutdeclaration2024.org/>, August 21-23, 2024
________________________________

**Comment by millerjeremya** [comment]
Making a google calendar invitation - how to invite nleanba?

**Comment by millerjeremya** [comment]
I don't have an email address for nleanba, so here are the details for an online meeting on Thursday morning.

Synospecies Authorities
Thursday, September 19 · 2:00 – 3:00pm
Time zone: Europe/Brussels
Google Meet joining info
Video call link: https://meet.google.com/iej-fivz-vha
Or dial: ‪(NL) +31 20 257 3161‬ PIN: ‪132 002 200‬#
More phone numbers: https://tel.meet/iej-fivz-vha?pin=6882431554651

**Comment by millerjeremya** [comment]
Sorry, Thursday at 14h is no longer available for me. nleanba, can you propose another time when you are available?

**Comment by myrmoteras** [comment]
Formica herculeana Linne

moved to a new genus by Fabricius, in animalia

Camponotus herculeanus (Linne)

**Comment by myrmoteras** [comment]
should missing base authorities be corrected in the annotation or in Synospecies?

**Comment by nleanba** [comment]
For a given name, we can find all different authorities given by treatments with
```sparql
PREFIX dwc: <http://rs.tdwg.org/dwc/terms/>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX treat: <http://plazi.org/vocab/treatment#>
SELECT ?name ?authority (GROUP_CONCAT(?treatment) AS ?treatments) WHERE {

  # Replace Name here as relevant
  BIND(<http://taxon-name.plazi.org/id/Animalia/Nanotyrannus_lancensis> AS ?name)


  ?tc treat:hasTaxonName ?name .
  GRAPH ?treatment {
    ?tc dwc:scientificNameAuthorship ?authority .
  }
}
GROUP BY ?name ?authority
```

For Nanotyrannus lancecnis we get these two outliers:
- "Gilmore, 1946" from "https://treatment.plazi.org/id/03A1879DFFD0FFE95E7FF664FEA6A618"
- "(Bakker, Williams, & Currie, 1988)" in "https://treatment.plazi.org/id/A56387B0FF8B704CB2BBFC9392CCFB4A"

I will make a list for the other names

**Comment by nleanba** [comment]
Tyrannosaurus synonyms with disagreements over authorities:

- Animalia/Aublysodon_lateralis — Cope, 1876 — 
    - https://treatment.plazi.org/id/D35787D0FF931578EF7CFB8DF7AEFBD6 
    - https://treatment.plazi.org/id/03BCF91DFFDE2E6AFE5EFCADB2D7F85F
- Animalia/Aublysodon_lateralis — (Cope, 1876) — 
    - https://treatment.plazi.org/id/1A7187CFFFC31777FDAEFA31E0075860

- Animalia/Aublysodon_mirandus — Leidy, 1868 — 
    - https://treatment.plazi.org/id/BB1087B9FFBCFFC1C4828274FB1A9E19 
    - https://treatment.plazi.org/id/D35787D0FF931578EF7DFCE5F788FC40 
    - https://treatment.plazi.org/id/7E4B202BFF9D1B29F942FE46DF2CD2C8 
    - https://treatment.plazi.org/id/1A7187CFFFC31777FDAEFA31E0075860 
    - https://treatment.plazi.org/id/6D5B047F403546462FE8FA8CFA0712EB 
    - https://treatment.plazi.org/id/03E0A360FFA05C1EDADD9048385E519B 
    - https://treatment.plazi.org/id/03B58786FFCEFFC148A6F813FD47DE5D 
    - https://treatment.plazi.org/id/03BD87E2D529FFF5FF7ED95F1BACF5C0
- Animalia/Aublysodon_mirandus — (Leidy, 1868) — 
    - https://treatment.plazi.org/id/03858796EA19FF998DEAFE40FBBBF8E6

- Animalia/Dynamosaurus_imperiosus — Osborn, 1905 — 
    - https://treatment.plazi.org/id/012B87EDFF98D8094DFA2581D9C7B0DB 
    - https://treatment.plazi.org/id/D35787D0FF90157AEF40FCE2FD1DFC4F 
    - https://treatment.plazi.org/id/03CD8786FF96FFED3A82F9C57DDCF68A
- Animalia/Dynamosaurus_imperiosus — (Osborn, 1905) — 
    - https://treatment.plazi.org/id/03EB9567FFE95F3BFCA27D28D2ACF9D4 
    - https://treatment.plazi.org/id/03E78790FFD4FF801145FE67FE7AFBFD

- Animalia/Laelaps_hazenianus — Cope, 1876 — 
    - https://treatment.plazi.org/id/820987CAAF49FFFD8121FCB0FD67FF63
- Animalia/Laelaps_hazenianus — (Hay, 1902) — 
    - https://treatment.plazi.org/id/1A7187CFFFC31777FDAEFA31E0075860

- Animalia/Laelaps_incrassatus — Cope, 1876 — 
    - https://treatment.plazi.org/id/820987CAAF4BFFFA8152FAA1FD64FCDF 
    - https://treatment.plazi.org/id/7128146C1F4D9401FEC310CDFE5FFDD3 
    - https://treatment.plazi.org/id/03BCF91DFFDE2E6BFE58F845B3A3FB84
- Animalia/Laelaps_incrassatus — (Cope, 1876) — 
    - https://treatment.plazi.org/id/1A7187CFFFC31777FDAEFA31E0075860
- Animalia/Laelaps_incrassatus — Cope, 1892 — 
    - https://treatment.plazi.org/id/03E3C81854415325FF5CC7DC329DF6D6

- Animalia/Nanotyrannus_lancensis — (Gilmore, 1946) — 
    - https://treatment.plazi.org/id/0F0287A7FFA3FFC97375FEE7FE1DF91D 
    - https://treatment.plazi.org/id/77323C29FFEBB41DFEBD9D05F87BFBDC 
    - https://treatment.plazi.org/id/012B87EDFF98D8094DFA2251D9D3B64B 
    - https://treatment.plazi.org/id/B97187EFFFC24A13FCE4FD5B7F0DFEAC 
    - https://treatment.plazi.org/id/038A657CFFC8FFF5FE14FD7F3415F96D 
    - https://treatment.plazi.org/id/03858796EA19FF998DEAFE40FBBBF8E6 
    - https://treatment.plazi.org/id/03DC7E37F05AFFA3FF49FE18FBD9FA1D 
    - https://treatment.plazi.org/id/03E8486CFFE30E52FFBE5400FBC3FE74
- Animalia/Nanotyrannus_lancensis — Gilmore, 1946 — 
    - https://treatment.plazi.org/id/03A1879DFFD0FFE95E7FF664FEA6A618
- Animalia/Nanotyrannus_lancensis — (Bakker, Williams, & Currie, 1988) — 
    - https://treatment.plazi.org/id/A56387B0FF8B704CB2BBFC9392CCFB4A

- Animalia/Struthiomimus_altus — Osborn, 1917 — 
    - https://treatment.plazi.org/id/D35787D0FF90157AEF43FC8EFD33FCFB
- Animalia/Struthiomimus_altus — (Lambe, 1902) — 
    - https://treatment.plazi.org/id/90598799FF1AFFB5FE89A36A5220F8A7 
    - https://treatment.plazi.org/id/90598799FF18FFA0FEFEA1CF50B5FD83

- Animalia/Stygivenator_molnari — Olshevsky, Ford, & Yamamoto, 1995 — 
    - https://treatment.plazi.org/id/A56387B0FF8B704CB5D0FB249308FAA5
- Animalia/Stygivenator_molnari — (Paul, 1988) — 
    - https://treatment.plazi.org/id/03858796EA19FF998DEAFE40FBBBF8E6 
    - https://treatment.plazi.org/id/03B9306BF077ED06A1F534A34142E177

- Animalia/Tyrannosaurus_rex — Osborn, 1905 — 
    - <Many>
- Animalia/Tyrannosaurus_rex — (Osborn, 1905) — 
    - https://treatment.plazi.org/id/D32F092B552A904C4689FD6C127AF072 
    - https://treatment.plazi.org/id/03D567223624FF7984DEFCDFF9A2F9D0

**Comment by myrmoteras** [comment]
Hi @nleanba thanks for this- have all of these couples an error - didn't we calculate 6 or 7 couples max? 

Just to be clear, all the new combinations, eg the ones with () need to have a basionym?

**Comment by nleanba** [comment]
In most cases, the treatments without "()" are missing the `baseAuthorityName` /-`Year` or have it wrongly as `authorityName` /-`Year`.

This is not always the case, for the two Tyrannosaurus Rex (Osborn 1905) treatments it probably should be authority instead of base authority (to bring it in agreement with the many other treatments which list Osborn as authority, and not as base-authority

**Comment by nleanba** [comment]
and
Animalia/Nanotyrannus_lancensis — (Bakker, Williams, & Currie, 1988) —
https://treatment.plazi.org/id/A56387B0FF8B704CB2BBFC9392CCFB4A

should have Bakker... as authority, and Gilmore as base-authority as far as I can tell

**Comment by nleanba** [comment]
To decide which is the correct base-authority/authority we must compare with the other treatments

**Comment by nleanba** [comment]
For example, I think that for Laelaps incrassatus, correct is
- Authority: Cope, 1876
- base-authority: None

![Screenshot_20240925_221056_Firefox.png](https://github.com/user-attachments/assets/72a5bed9-df02-4066-9e6c-e15580cf574c)

**Comment by nleanba** [comment]
I wonder why this annotation change has not been pushed to the treatments-xml repo though, it would have been correct as far as I can tell

> Thanks for this example. I think it's a useful approach to work through. 
> So the thing that jumps out at me has to do with Treatment 03A1879DFFD0FFE95E7FF664FEA6A618
> You provide these annotations in the example
> 
>  <taxonomicName authorityName="Gilmore" authorityYear="1946" ...>Nanotyrannus</taxonomicName>
> →
> 
>  <http://taxon-concept.plazi.org/id/Animalia/Nanotyrannus_lancensis_Gilmore_1946>
>      dwc:authorityName "Gilmore" ;
>      dwc:authorityYear "1946" ;
>      dwc:scientificNameAuthorship "Gilmore, 1946" .
> 
> 
> This was indeed a tricky one to annotate. It is a genus level treatment (new genus) but of a monotypic genus, so it works kind of like a species level treatment. So I'm changing the authorityName and authorityYear annotations into baseAuthorityName and baseAuthorityYear annotations. How does that effect the result on Synospecies? 
> 
> ![image](https://github.com/user-attachments/assets/5b7073ca-c80e-414c-b866-5bbd0f353842)
> ![image](https://github.com/user-attachments/assets/d30ecdef-ca48-4452-9514-e17eb72c7a34)
> 
> 
>

**Comment by nleanba** [comment]
See also https://github.com/factsmission/synospecies/issues/134#issuecomment-2823760761

(Im closing the other issue in favor of this one)
